src/index.pug

```diff
@@ -83,9 +84,9 @@ block content
                 /* 4 */ return true;
                 ```
 
-                We call those modifications mutants. After the mutants have been found, they are applied one by one and your tests will be executed against them. 
-                If at least one of your tests fail, we say the mutant is *killed*. That's what we want! If no tests fail, it *survived*. 
-                The better your tests, the less mutants survive.
+                We call those modifications mutants. After the mutants have been found, they are applied one by one and your tests are executed against them. 
+                If at least one of your tests fail, we say the mutant is *killed*. That's what we want! If no test fail, it *survived*. 
```

Comment:
I don't think the change is correct now. It should be "If no test fails"

Reply:
indeed. Haven't noticed